code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -47,3 +47,34 @@
 	embedding = list(embed_chance=100, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
 	wound_falloff_tile = -5
 	shrapnel_type = /obj/item/ammo_casing/harpoon
+
+// Rebar (Rebar Crossbow)
+/obj/projectile/bullet/rebar
+	name = "rebar"
+	icon_state = "rebar"
+	damage = 40
+	stamina = 20
+	speed = 0.4
+	dismemberment = 1 //because a 1 in 100 chance to just blow someones arm off is enough to be cool but also not enough to be reliable
+	armour_penetration = 12
```

Comment:
There is no point in armor pen in values that aren't in multiples of 5. I'd just round this down to 10.

Reply:
Fixed